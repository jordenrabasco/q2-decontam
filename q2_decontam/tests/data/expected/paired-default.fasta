>cbd46c06c1ddceeb8d856468c1b549c8
TACGTAGGGTGCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCCGAGCTTGCAGCCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGGAGCAAACAGG
>c15fd6a65f4f8d575c2cb4cfe2a659a5
TACGTAGGGGGCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCGTGTAGGCGGTCTGGTAAGTCTGCCGTGAAAACCCGGGGCTCAACCCCGGGCGTGCGGTGGATACTGCCAGGCTAGAGGGTGGTAGAGGCGAGTGGAATTCCCGGTGTAGCGGTGAAATGCGCAGATATCGGGAGGAACACCAGTAGCGAAGGCGGCTCGCTGGGCCATTCCTGACGCTGAGACGCGAAAGCTAGGGGAGCGAACAGG
>854614d933d994aecc1d1b7bbc24e675
TACGTAGGGTGCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCTGTTACGTCGGCTGTGAAAACTCGGGGCTCAACCCTGAGCCTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGG
>7925aae7da7d248eccdcd0ed0935ad61
TACGTAGGGGGCAAGCGTTGTCCGGAATCATTGGGCGTAAAGAGCGCGTAGGCGGTCCGGTAAGTCCATCGTGAAAGCCAAGGGCTCAACCCTTGGATGCCGGTGGATACTGTCGGGCTAGAGTCCGGAAGAGGCGAGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAAGAACACCTATGGCGAAGGCAGCTCGCTGGGACGGAACTGACGCTGAGGCGCGAAAGCGTGGGGAGCAAACAGG
>5de23fc8df270235a8958de416e2f944
TACGTAGGGGGCAAGCGTTGTCCGGATTCATTGGGCGTAAAGAGCTCGTAGGCGGCTTGGCAAGTCGGGTGTGAAAACTTCAGGCTCAACCTGGAGATGCCACCCGAGACTGCTATGGCTAGAGTCCGGTAGGGGACCACGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTAGCGAAGGCGGTGGTCTGGGCCGGCACTGACGCTGAGGAGCGAAAGCGTGGGTAGCAAACAGG
>27ae47cc0e39b6df154b6938d9848c1b
TACGTAGGGTGCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTTTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCTGAGCTTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGG
>acca130825261f8ae2e602bfe1865525
CACGTAGGGACCGAGCGTTGTCCGGAATCATTGGGCGTAAAGAGCGCGTAGGCGGCTATCCACGTCGGGTGTGAAAGCTCAGAGCTCAACTCTGAGAGGCCATTCGATACGGGATAGCTAGAGGCAGCTAGGGGAGAGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGCTCTCTGGGGCTGATCTGACGCTGAGGCGCGAAAGCTGGGGGAGCAAACAGG
>0018939bf02c32c9a63c42ade52cefad
TACGGAGGGTGCGAGCGTTAATCGGAATTACTGGGCGTAAAGGGCGCGTAGGTGGTTGGGTAAGTCGGGTGTGAAAGCCCTGGGCTTAACCTGGGAATGGCATTCGAGACTACCTATCTAGAGTCTGGTAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGTGTAGATATCGGAAGGAACACCAGTGGCGAAGGCGACTTCCTGGACCAAGACTGACGCTGAGGCGCGAAAGCGTGGGGATCAAACAGG
>31ca42f58709548b0c88ae9153bece35
TACGGAGGATGCAAGCGTTATCCGGATTTATTGGGTTTAAAGGGTGCGTAGGCGGCCTGTTAAGTCAGGGGTGAAAGGTTGCGGCTTAACCGCAGAATTGCCTTTGATACTGGCGGGCTTGAATAAAGTTGAGGTAGGCGGAATGTGGCAAGTAGCGGTGAAATGCATAGAGATGCCACAGAACACCGATTGCGAAGGCAGCTTACTAAGCTTTGATTGACGCTGAGGCACGAAAGCGTGGGGATCAAACAGG
>4ec5e8b7a8c0d46bebe446577daa0352
CACGTAGGGGGCGAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCCGTTCGCGTCGGATGTGAAAGCTCAAGGCTCAACCTTGAGAGGCCATTCGATACGGGACGGCTGGAGGCAGGCAGGGGAGAGTGGAATTCCTGGTGTAGCGGTGGAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGCTCTCTGGGCCTGTTCTGACGCTGAGGAGCGAAAGCTGGGGTAGCAAACAGG
>90179b7710e4f00cc4de907ada3d13a5
TACGTAGGGTCCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCCGTTAAGTCGGAAGTGAAATCTCAAGGCTCAACCTTGAAATTGCTTTCGATACTGGCGGACTTGAGGCAGGTAGGGGAAGGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCAGTGGCGAAGGCGGCCTTCTGGGCCTGTACTGACGCTGAGGAGCGAAAGCGTGGGGAGCAAACAGG
>bb0a57a999432eb92864d72d6284edae
TACGTAGGGACCAAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCTGTTCGCGTCGGATGTGAAAGCTCAGAGCTCAACTCTGAGAGGCCATTCGATACGGGATAGCTAGAGGTAGGTAGGGGAGATCGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGATCTCTGGGCCTTACCTGACGCTGAGGAGCGAAAGCTGGGGGAGCAAACAGG
>d324c95a5966d689591653062ca99d03
TACGTAGGGGGCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCGTGTAGGCGGCCTGGTAAGTCTGCTGTGAAACCTTGGGGCTCAACCCCAAGCGTGCAGTGGATACTGCTAGGCTAGAGGATGGTAGAGGCGAGTGGAATTCCCGGTGTAGCGGTGAAATGCGCAGATATCGGGAGGAACACCAGTAGCGAAGGCGGCTCGCTGGGCCATTCCTGACGCTGAGACGCGAAAGCTAGGGGAGCGAACAGG
>3587ee8f8d9cefe13b27e5559b8e1b58
TACGTAGGGGGCAAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCCTGGTGAGTCGGGTGTGAAAGCCCGAGGCTCAACCTCGGAATTGCATTCGATACTGCTGGGCTTGAGGCAGGTAGGGGAGGATGGAATTCCCGGTGTAGCGGTGGAATGCGCAGATATCGGGAGGAACACCTGCGGCGAAGGCGGTCCTCTGGGCCTGTCCTGACGCTGAGGAGCGAAAGCGTGGGGAGCGAACAGG
>868e0aaa76b7fda3a2d8845889ca8f2b
TACGGAGGGTGCGAGCGTTAATCGGAATTACTGGGCGTAAAGGGCGCGTAGGCGGTTGGGTAAGTCGGATGTGAAAGCCCTGGGCTTAACCTGGGAATGGCATTCGAGACTACCTATCTAGAGTCTGGTAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGTGTAGATATCGGAAGGAACACCAGTGGCGAAGGCGACTTCCTGGACCAAGACTGACGCTGAGGCGCGAAAGCGTGGGGATCAAACAGG
>f2fd9d7b74d80ad0cc6f3439e6325d27
TACGGAGGGTGCGAGCGTTGTCCGGAATCACTGGGCGTAAAGGGCGCGTAGGCGGCCTGATAAGTAGGGGGTGAAATCCTGCGGCTTAACCGCAGGGCTGCCTTCTAAACTGTCAGGCTCGAGCACAGTAGAGGCAGGTGGAATTCCCGGTGTAGCGGTGGAATGCGTAGAGATCGGGAAGAACATCAGTGGCGAAGGCGGCCTGCTGGGCTGTTGCTGACGCTGAGGCGCGACAGCGTGGGGAGCAAACAGG
>f01fcb3165881248269b22cf5ff01ebe
GACGTAGGGCGCGAGCGTTGTCCGGATTTATTGGGCGTAAAGGGCTCGTAGGCGGCTCGTCGCGTCTGTCGTGAAAGGCTCCGGCTTAACCGGGGTTTTGCGGTGGATACGGGCGGGCTTGAGGCTAGTAGGGGAGATCGGAATTCTACGTGTAGCGGTGAAATGCGCAGATATGTGGAGGAACACCGGTGGCGAAGGCGGATCTCTGGGCTGGTTCTGACGCTGAGGAGCGAAAGCATGGGGAGCGAACAGG
>e16d86b7fe770de88e7e8fd0fb6c8b5a
TACAGAGGGTGCGAGCGTTAATCGGAATTACTGGGCGTAAAGCGCGCGTAGGCGGCTTTGCAAGTCGGGTGTGAAATCCCCAGGCTTAACCTGGGAACTGCATTCGAGACTGCATTGCTAGAGTATGGGAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGCGTAGATATCGGAAGGAACATCAGTGGCGAAAGCGACTTCCTGGACCAATACTGACGCTCATGTGCGAAAGCGTGGGGAGCAAACAGG
