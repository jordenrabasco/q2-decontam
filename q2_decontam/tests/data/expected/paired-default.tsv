# Constructed from biom file
#OTU ID	BAQ2420.2	BAQ3473.2	BAQ3473.3	BAQ4166.2	BAQ4166.3	PAP130.2	YUN1005.3	YUN1242.1	YUN1242.3	YUN3153.3
cbd46c06c1ddceeb8d856468c1b549c8	0.0	0.0	0.0	0.0	0.0	0.0	0.0	30.0	0.0	0.0
c15fd6a65f4f8d575c2cb4cfe2a659a5	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	28.0	0.0
854614d933d994aecc1d1b7bbc24e675	0.0	0.0	0.0	0.0	0.0	0.0	22.0	0.0	0.0	0.0
7925aae7da7d248eccdcd0ed0935ad61	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	20.0
5de23fc8df270235a8958de416e2f944	0.0	0.0	0.0	0.0	0.0	0.0	18.0	0.0	0.0	0.0
27ae47cc0e39b6df154b6938d9848c1b	0.0	0.0	0.0	0.0	0.0	0.0	17.0	0.0	0.0	0.0
acca130825261f8ae2e602bfe1865525	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	16.0	0.0
0018939bf02c32c9a63c42ade52cefad	15.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
31ca42f58709548b0c88ae9153bece35	0.0	0.0	11.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
4ec5e8b7a8c0d46bebe446577daa0352	0.0	0.0	0.0	0.0	0.0	0.0	11.0	0.0	0.0	0.0
90179b7710e4f00cc4de907ada3d13a5	0.0	0.0	0.0	0.0	0.0	0.0	0.0	11.0	0.0	0.0
bb0a57a999432eb92864d72d6284edae	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	11.0
d324c95a5966d689591653062ca99d03	0.0	0.0	0.0	0.0	0.0	0.0	0.0	8.0	0.0	0.0
3587ee8f8d9cefe13b27e5559b8e1b58	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	6.0	0.0
868e0aaa76b7fda3a2d8845889ca8f2b	0.0	0.0	5.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
f2fd9d7b74d80ad0cc6f3439e6325d27	0.0	0.0	0.0	0.0	5.0	0.0	0.0	0.0	0.0	0.0
f01fcb3165881248269b22cf5ff01ebe	0.0	0.0	0.0	0.0	0.0	5.0	0.0	0.0	0.0	0.0
e16d86b7fe770de88e7e8fd0fb6c8b5a	0.0	0.0	0.0	0.0	4.0	0.0	0.0	0.0	0.0	0.0